rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2431,3 +2431,81 @@ RocalTensor rocalNormalDistribution(RocalContext p_context,
     }
     return output;
 }
+
+std::pair<RocalTensor, RocalTensor> ROCAL_API_CALL
+rocalNonSilentRegion(
+        RocalContext p_context,
+        RocalTensor p_input,
+        bool is_output,
+        float cutoff_db,
+        float reference_power,
+        int reset_interval,
+        int window_length) {
+    Tensor* output1 = nullptr;
+    Tensor* output2 = nullptr;
+    if ((p_context == nullptr) || (p_input == nullptr))
+        ERR("Invalid ROCAL context or invalid input tensor")
+    TensorList output_tensors;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    try {
+        RocalTensorDataType tensor_data_type = RocalTensorDataType::INT32;
+        unsigned number_of_dims = 2;
+        std::vector<size_t> dims1(number_of_dims, 1);
```

Comment:
Can we not say as std::vector<size_t> dims1 = {batch_size, 1};

That will remove number_of_dims variable right

Reply:
done